docs/Tutorials/GettingStarted/Tutorial.md

```diff
@@ -0,0 +1,226 @@
+# Getting Started with F´
+
+Prerequisites: [https://github.com/nasa/fprime/blob/master/INSTALL.md](https://github.com/nasa/fprime/blob/master/INSTALL.md)
+
+This tutorial will walk the user through the basic usage of the F´ development suite. This assumes that the user has run
+through the above installation guide, and has a working F´ installation.
+
+The goal of this Tutorial is to build familiarity with the F´environment and tools. It does not walk the user through
+creation of new F´ items, but rather users the existing F´ `Ref` example as a place to operate the tools, learn the
+environment, and learn F´.
+
+The full development process is covered in the [Math Component Tutorial](../MathComponent/Tutorial.md), which will build
+on the knowledge learned here.
+
+## F´ Terminology
+
+This section will cover basic terminology used in this and other tutorial with respect to F´.  It may be used as a
+reference for keywords that have specific meaning in an F´ project.
+
+**Port:** a port represents a connection between *Components*. These act as communication channels in F´. This allows
+*Components* to interact with each other. A set of *Ports* acts as a defined interface to a *Component*. 
+
+**Component:** a single unit in F´. This represents one conceptual portion of an F´ deployment.  Typically encapsulates
```

Comment:
This description is awkward. Can we change it to: "the location of program behavior. Components execute the software logic of the system..."

Reply:
Ok. Will do!

Reply:
Should be fixed.  Please take a look.